patterns/2-structured/discover-your-innersource.md

```diff
@@ -43,23 +26,26 @@ People don't bother looking for internally developed solutions - they might not
     - might not be maintained
     - might have poor reusability
     - if someone put out a SW internally, the expectation is that they wouldn't have time to support it (vs. open source options)
+* Difficulty in adding content to the search index makes it hard for people to find anything
 
 ## Solution
 
 Make it easy to find the reusable code.
 
 * Pull in Repo names, descriptions and README.md files into the search engine assuming that a search engine exists. Note: a one-stop-shop kind of search engine for all relevant communication and documentation is difficult to attain. Even when using Github's enterprise offering, people often deploy additional systems like wikis to host content orthogonal to what is in the code repositories, slack channels (or IRC), mailing lists (or even nntp servers), some place to store stuff written down in office formats, search over personal e-mail etc. Several of these systems come with search built-in, but integrating this content in one search box or even just deploying a federated search engine across all sources often doesn't come off-the-shelf.
+* Consider using existing tools:
+  * GitHub Enterprise and npmjs have built-in searches on meta-data
+  * PayPal's Agora project (planned to be open sourced) addresses similar challenges with data source integration
```

Comment:
I could not find a mention of the PayPal Agora project anywhere.
So either they renamed it, or they never open sourced it.

We could potentially ask the ex-PayPal people for help here.

Or we remove this entire sentence, if it is not adding helpful info to this pattern.

Reply:
Sounds good to me!

Reply:
Thanks for the link Russ.

They even mention the ISC in the Readme. But yeah, archived in 2020.

This could be one of those software assets that we port to our GitHub org. However probably not worthwhile if there are no users of the software.

Aside: maybe I should search GitHub for so mentioned of the InnerSource Commons.

Reply:
It is mostly the search engine.

Reply:
Silona left, who was championing the effort.